packages/react/src/components/CustomerContribution/CustomerFeedbackModal.tsx

```diff
@@ -82,7 +82,7 @@ export const CustomerFeedbackModal: React.FC<Props> = React.memo(({ visible, set
                       })
                     );
                   } else {
-                    console.error("Failed to send customer feedback: ", response.errorValue);
+                    window.electron.writeReactErrLog("Failed to send customer feedback - PA UI: " + response.errorValue)
```

Comment:
Using this format to write the logs will help the lambda use it further:
https://github.com/aws/porting-assistant-dotnet-ui/blob/33c64aaf78d89612bb0d1c443c2a40659a17a81d/packages/csharp/PortingAssistant/PortingAssistant.Api/Program.cs#L34
You can check in the above link how its is logged.